Rakefile

```diff
@@ -17,6 +17,7 @@ require 'rspec/core/rake_task'
 require 'open3'
 require 'os'
 require 'json'
+require 'fileutils'
```

Comment:
minor: I see that you are taking the dependency on `fileutils` external gem for `rm_rf` then don't you need to add it in the Gemfile for development group? Secondly, I'd prefer not taking dependency if you can re-use the commands that we are already using to delete.

Reply:
Fileutils is a better and cleaner approach than the current approach. Also, it is a standard library in ruby. So, you do not want explicit invocations in Gemfile.